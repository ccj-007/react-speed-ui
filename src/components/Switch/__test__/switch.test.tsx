import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Switch } from '../../index';

describe('test Switch component', () => {
  it('should render', () => {
    const { asFragment } = render(<Switch />);
    expect(asFragment()).toMatchSnapshot();
  });
});
