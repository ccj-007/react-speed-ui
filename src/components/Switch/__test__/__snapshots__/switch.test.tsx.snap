// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Switch component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-switch css-11ly021-Switch"
  >
    <div
      class="speed-switch-circle css-163r71z-Switch"
    />
  </div>
</DocumentFragment>
`;
